pkg/cmd/openshift-kube-apiserver/kubeadmission/register.go

```diff
@@ -43,8 +44,7 @@ var (
 	SkipRunLevelOnePlugins = sets.NewString(
 		"authorization.openshift.io/RestrictSubjectBindings",
 		"autoscaling.openshift.io/ClusterResourceOverride",
-		"autoscaling.openshift.io/RunOnceDuration",
```

Comment:
this relies on projects (project cache for some reason), which aren't available before openshift-apiserver comes up.  we should probably keep it.

Reply:
will add back in.